src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsFlyout.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Menu } from './MediaSettingsMenu';
+import './MediaSettingsFlyout.scss';
+
+export type Props = {
+    children: React.ReactNode;
+    className?: string;
+    menu: Menu;
+    menuSelectors: Record<string, string>;
+};
+
+const SETTINGS_MENU_PADDING = 18;
+const SETTINGS_MENU_PADDING_SCROLL = 32;
+const SETTINGS_MENU_MAX_HEIGHT = 210;
```

Comment:
These values seem a bit arbitrary. Are they coming from the existing settings controls?

Reply:
Yeah